LiveStream/LiveStream.h

```diff
@@ -0,0 +1,19 @@
+//
+//  LiveStream.h
+//  LiveStream
+//
+//  Created by Brandon Williams on 1/5/17.
+//  Copyright © 2017 Kickstarter. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+//! Project version number for LiveStream.
+FOUNDATION_EXPORT double LiveStreamVersionNumber;
+
+//! Project version string for LiveStream.
+FOUNDATION_EXPORT const unsigned char LiveStreamVersionString[];
+
+// In this header, you should import all the public headers of your framework using statements like #import <LiveStream/PublicHeader.h>
+
```

Comment:
is this file a mistake? objc? if not, remove Brando credit at the top and instructions at the bottom.